src/test/java/core/basesyntax/UserRegistrationTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class UserRegistrationTest {
+    private static final String SUCCESS_LOGIN = "SUCCESS_LOGIN";
+    private static final String SUCCESS_PASSWORD = "SUCCESS_PASSWORD_123";
+    private static final Integer SUCCESS_AGE = 19;
+    private static final String NOT_SUCCESS_LOGIN = "short";
+    private static final String NOT_SUCCESS_PASSWORD = "short";
+    private static final Integer NOT_SUCCESS_AGE = 15;
+    private static final User user = new User();
+    private RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void setUserValidParams() {
+        user.setLogin(SUCCESS_LOGIN);
+        user.setPassword(SUCCESS_PASSWORD);
+        user.setAge(SUCCESS_AGE);
+    }
+
+    @AfterEach
+    void setValidUserParams() {
+        user.setLogin(SUCCESS_LOGIN);
+        user.setPassword(SUCCESS_PASSWORD);
+        user.setAge(SUCCESS_AGE);
+        registrationService.removeByLogin(SUCCESS_LOGIN);
+    }
+
+    @BeforeEach
+    void setStorageDao() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void allUserValuesValid_OK() {
+        assertEquals(user, registrationService.register(user));
+    }
+
+    @Test
+    void nullUser_NotOK() {
+        assertNull(null);
```

Comment:
The `nullUser_NotOK` test case has an incorrect assertion. Instead of `assertNull(null)`, it should test the behavior of the `register` method with a `null` user. Update this test to call `registrationService.register(null)` and expect an exception.